### ISSUE TITLE
Program don't work at all

### ISSUE THREAD

**Issue by cybergollum** [HearthStats/HearthStats.net-Uploader#673]
Hello.

I had updated Java (6->8), then started Hearthstats.exe but visually nothing were happened. I tried again and waited watching at Task Manager. There appeared "javaw.exe" for 5-10 seconds and then disappeared. At folder where executable located appeared empty "log.txt" and folders "tmp" and "cache" both empty too.

I need to add that currently I use WinXP SP3 and so... is this my problem? If so, is there any way I could run program?

**Comment by trigun0x2** [comment]
@cybergollum as of now, I don't think we'll be supporting WinXP. I suggest using: https://github.com/Epix37/Hearthstone-Deck-Tracker

As it also uploads to HearthStats!

**Comment by cybergollum** [comment]
Sigh. 

Yes, I tried it too and unfortunately it didn't run. But in other way - at least it raised exception kinda "....Hearthstone Deck Tracker.exe isn't Win32 application". 
Whatever. I just hoped that I could convince developer(s) if it the only one thing needed to change - at Project Properties change target from win7, .net 4.5 to "winxp". Probably I called parameters wrong but I am sure you understood me :)

Anyway, big thanks to you (not only for quick answer but for all other your work :)

PS: before I will finally upgrade PC I still will be continuing manually add card info in online services like I was doing it for more than a year already. And by the way I found one program that is working for me. It hasn't great functionality but still... it grab all info automatically. So it will be great addition to my manual insertion.

Program called "Track-o-Bot" if you are interested. I know you heard about it.
